static-site/src/pages/users.jsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import styled from "styled-components";
+import UserDataWrapper from "../layouts/userDataWrapper";
+import NavBar from '../components/nav-bar';
+import { Logos } from "../components/logos";
+
+const UserPage = (props) => {
+  const { user } = props;
+
+  const LoggedIn = () => (
+    <p>
+      You&apos;re logged in as <strong>{user.username}</strong>.<br />
+      You have access to the following groups:
```

Comment:
Add some text here explaining that the following are only private groups (e.g. `seatteflu` will be a public group and won't show up here as it won't have a corresponding cognito group attached to the user). E.g. "You have access to the following private nextstrain groups, which each contain a collection of datasets and/or narratives:"